### ISSUE TITLE
Force double door not changing checkbox

### ISSUE THREAD

**Issue by SeanRadvan** [CCDevNelson/Snap-Updates#322]
![image](https://user-images.githubusercontent.com/54151330/110661608-fbc9c200-8181-11eb-8ecd-19c634cc44d9.png)
increasing the width of a pard correctly forces double doors over 24" but isn't updating the checkbox value to reflect this.  Appears to be working correctly on placement.  
*note the box wont change anything because double doors are forced, the box should just reflect this.

**Comment by TylerStandage** [comment]
Thanks @SeanRadvan .  @sarahhightower where the opening of the section is large enough that double doors are forced, do you think designers would prefer to have the box "checked", or would it be better to remove that line/checkbox altogether since the option doesn't exist anyway?

**Comment by TylerStandage** [comment]
@RyanMDev we have a number of issues related to file saving. I'd like to get as many as possible resolved for 2.0 release. I'll get the info together into one place so we can talk abt it on Friday.

**Comment by TylerStandage** [comment]
@JJanneh @SeanRadvan I agree with @sarahhightower that we should probably leave it there. Note that 24 inches is the limit for a section width for a single door, so anything wider than that would be an exception that should be approved and discussed in detail. Is it possible to have a pop-up note when hovering over the check box that describes the rule so that it's not too confusing to the designer?

**Comment by JJanneh** [comment]
@TylerStandage @ClassyTeddy 
I believe it's possible. I just would have to touch base. Request to have a pop-up note when hovering over the check box that describes the rule so that it's not too confusing to the designer. Stating Note that 24 inches is the limit for a section width for a single door, so anything wider than that would be an exception that should be approved and discussed in detail.

**Comment by ClassyTeddy** [comment]
So @JJanneh and @TylerStandage, having a popup show up while in the prompt menu unfortunately is not an option. We can however create a new line and display Text that reads similar to how the popup would read, and it could only show up if the new 'Double Door Auto Switch' prompt was changed from the default of 24"
